### DISCUSSION TITLE
BT-14 in subtypes 7-24

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#901] [Ideas]
According to the fields-json, either BT-15 or BT-615 is mandatory in subtypes 7-24.

However, BT-15 or BT-615 is only displayed (not forbidden) if BT-14 is filled in. Shouldn't BT-14 then be mandatory in the corresponding subtypes? (Currently it's only mandatory in: 16, 17, 19, 23, 24)

**Comment by YvesJo** [comment]
Hi,
Procurement Documents (BG-101) is only required for the forms 16, 17, 19, 23 & 24, and BT-14 can not be made mandatory for other forms as it would then require the existence of other Business Terms of this Business Group.
BT-15 and BT-615 share a same XML element and may only be distinguished with the value of BT-14

**Reply by JanEmde** [reply]
Got it. Thank you.
